Gopkg.toml

```diff
@@ -15,6 +15,11 @@ required = [ "k8s.io/code-generator/cmd/client-gen", "k8s.io/gengo/types" ]
   non-go = false
   unused-packages = false
 
+[[prune.project]]
+  name = "github.com/openshift/library-go"
+  non-go = false
```

Comment:
update the commit message with reason for prune exception

Reply:
Done.